composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "inchoo/productFAQ",
+    "description": "N/A",
+    "config": {
+        "sort-packages": true
+    },
+
+    "type": "magento2-module",
+    "license": [
+        "OSL-3.0",
+        "AFL-3.0"
+    ],
+    "autoload": {
+        "files": [
+            "registration.php"
+        ],
+        "psr-4": {
+            "Inchoo\\ProductFAQ\\": ""
+        }
+    }
+}
```

Comment:
ovako nekako treba izgledati composer.json i uključivati module koje se koriste unutar tvoga (uključujući i framework, backend)

{
  "name": "inchoo/module-test",
  "description": "Magento 2 Test integration",
  "type": "magento2-module",
  "version": "1.0.0",
  "license": [
    "OSL-3.0"
  ],
  "require": {
    "php": "~7.1.3||~7.2.0||~7.3.0",
    "magento/framework": "*",
    "magento/module-backend": "*",
    "magento/module-sales": "*",
    "magento/module-ui": "*",
    "inchoo/module-sales-grid-improved": "*",
    "inchoo/module-print-templates": "*",
    "inchoo/module-related-products-replacement": "*",
    "magento/module-catalog": "*",
    "magento/module-eav": "*",
    "magento/module-catalog-url-rewrite": "*",
    "magento/module-configurable-product": "*"
  },
    "autoload": {
        "files": [
            "registration.php"
        ],
        "psr-4": {
            "Inchoo\\Test\\": ""
        }
    },
    "authors": [
        {
            "name": "Inchoo",
            "email": "backend@inchoo.net",
            "homepage": "https://inchoo.net/",
            "role": "Developer"
        }
    ]
}

Reply:
prepravi da bude tako poredano i ima require onoga što koristi

Reply:
dodan require i poredano prema primjeru